addons/website/views/snippets/s_numbers.xml

```diff
@@ -4,32 +4,28 @@
 <template name="Numbers" id="s_numbers">
     <section class="s_numbers o_cc o_cc1 pt80 pb80">
         <div class="container">
-            <div class="row">
-                <div class="col-lg-4">
+            <div class="row s_nb_column_fixed">
+                <div class="col-lg-5">
                     <h3 class="mb-3 h4">Key Metrics of<br/>Company's Achievements</h3>
                     <p class="lead">Analyzing the numbers behind our success: <br class="d-none d-xxl-inline"/>an in-depth look at the key metrics driving our company's achievements</p>
                 </div>
-                <div class="col-lg-7 offset-lg-1">
-                    <div class="row">
-                        <div class="col-lg-4 pt24 pb24 text-center" data-name="Number Box">
-                            <p class="lead">
-                                <span class="s_number display-1-fs">12k</span><br/>
-                                <span class="h5-fs">Useful options</span>
-                            </p>
-                        </div>
-                        <div class="col-lg-4 pt24 pb24 text-center" data-name="Number Box">
-                            <p class="lead">
-                                <span class="s_number display-1-fs">45%</span><br/>
-                                <span class="h5-fs">More leads</span>
-                            </p>
-                        </div>
-                        <div class="col-lg-4 pt24 pb24 text-center" data-name="Number Box">
-                            <p class="lead">
-                                <span class="s_number display-1-fs">8+</span><br/>
-                                <span class="h5-fs">Amazing pages</span>
-                            </p>
-                        </div>
-                    </div>
+                <div class="col-lg-2 offset-lg-1">
+                    <p style="text-align: center;">
+                        <span class="display-1-fs">12k</span><br/>
```

Comment:
You removed the `s_number` classes but I see many themes xpathing it. Should not you just keep it?